staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -455,7 +566,7 @@ func (meal *cfgMeal) digestFlowSchemasLocked(newFSs []*flowcontrol.FlowSchema) {
 		//
 		// TODO: consider not even trying if server is not handling
 		// requests yet.
-		meal.presyncFlowSchemaStatus(fs, !goodPriorityRef, fs.Spec.PriorityLevelConfiguration.Name)
+		meal.presyncFlowSchemaStatus(fs, meal.cfgCtlr.foundToDangling(goodPriorityRef), fs.Spec.PriorityLevelConfiguration.Name)
```

Comment:
`foundDangling` is related to rate limiting?  Is this function doing more than simply inverting the `goodPriorityRef`?

Reply:
The name is "found _to_ dangling".  It maps the boolean indicating whether the referenced PriorityLevelConfiguration object was found to the boolean value that should be put in the "Dangling" condition, and in the current release it simply returns the inverse of what it is given.  It is a variable so that the integration test can exercise a mixture of behaviors.